server/middleware/headers.go

```diff
@@ -16,18 +16,35 @@ package middleware
 
 import (
 	"context"
+	"fmt"
+	"time"
 
+	api "github.com/tigrisdata/tigris/api/server/v1"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/metadata"
 )
 
 var OutgoingHeaders = metadata.New(map[string]string{})
 
+const (
+	OutboundCookieHeader = "Set-Cookie"
```

Comment:
What's the benefit of defining these constants? These are implicitly constants as these are never going to change

Reply:
This helps to avoid typos when used in multiple places in the code.

Reply:
used api.SetCookie